src/penn_chime/models.py

```diff
@@ -147,17 +147,29 @@ def __init__(self, p: Parameters):
         self.daily_growth_rate_t = get_growth_rate(self.doubling_time_t)
 
     def run_projection(self, p):
```

Comment:
I might pass the (beta, n_day) pairs into `run_projection(p, *args)`, instead of using state. The call site knows better if it is pre or post fit, and how many of these pairs are needed.

If I was going to use state, then I would store the pairs as in a list:
```
self.policies = [beta1, n_days1, beta2, n_days2, beta3, n_days3, beta4, n_days4...]
...
        self.raw_df = sim_sir_df(
            self.susceptible,
            self.infected,
            p.recovered,
            self.gamma,
            -self.i_day,
            *self.policies)
```

... to avoid having to rewire `run_projection` when extending the ui to allow multiple (social_distancing, date) pairs.

Ideally, the n_days input from the UI input would be merely a UI setting for the graphs, and gen_sir would simply run until i < 0.5. This would avoid complicating the model total days computations

Reply:
Opened #447.